internals/manifestworkSync/adapters/manifestwork.go

```diff
@@ -0,0 +1,67 @@
+package adapters
+
+import (
+	"context"
+	"fmt"
+	rcsv1alpha1 "github.com/dana-team/container-app-operator/api/v1alpha1"
+	events "github.com/dana-team/rcs-ocm-deployer/internals/utils/sharedEvents"
+	"github.com/go-logr/logr"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/record"
+	workv1 "open-cluster-management.io/api/work/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	NamespaceManifestWorkPrefix = "mw-create-"
+	CappNameKey                 = "dana.io/capp-name"
+	CappNamespaceKey            = "dana.io/capp-namespace"
```

Comment:
To be consistent with the API, we should probably change the group here to `rcs.dana.io`